src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-add-content.js

```diff
@@ -0,0 +1,103 @@
+(function (global) {
+    if (global.CKEDITOR.plugins.get('ezaddcontent')) {
+        return;
+    }
+
+    /**
+     * Creates a given HTMLElement
+     *
+     * @method createElement
+     * @return HTMLElement
+     */
+    const createElement = (doc, tagName, content, attributes) => {
+        const element = doc.createElement(tagName);
+
+        element.setAttributes(attributes);
+        element.setHtml(content ? content : "<br>");
```

Comment:
Why `<br />`?

Reply:
AlloyEditor requires some content if you add a new item.